test/parallel/test-windows-failed-heap-allocation.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
+const common = require('../common');
+
+// This test ensures that an out of memory error exits with code 134 on Windows
+
+if (!common.isWindows) return common.skip('Windows-only');
+
+const assert = require('assert');
+const { exec } = require('child_process');
+
+if (process.argv[2] === 'heapBomb') {
+  // heap bomb, imitates a memory leak quickly
+  const fn = (nM) => [...Array(nM)].map((i) => fn(nM ** 2));
+  fn(2);
+}
+
+// --max-old-space-size=3 is the min 'old space' in v8, explodes fast
```

Comment:
Small nit: `v8` -> `V8`. This will help distinguish between Node.js 8 and V8 JavaScript engine.